daft/io/_parquet.py

```diff
@@ -26,6 +26,7 @@ def read_parquet(
     file_path_column: Optional[str] = None,
     hive_partitioning: bool = False,
     coerce_int96_timestamp_unit: Optional[Union[str, TimeUnit]] = None,
+    ignore_error: bool = False,
```

Comment:
`ignore_error` is pretty ambiguous because there are many different scenarios that can produce errors during file reading that we don't want to ignore. `ignore_corrupt_files` is probably a better fit.

Also we should add these new parameters to the docstring below.